services/server/config/config.go

```diff
@@ -164,8 +164,10 @@ type CgroupConfig struct {
 
 // ProxyPlugin provides a proxy plugin configuration
 type ProxyPlugin struct {
-	Type    string `toml:"type"`
-	Address string `toml:"address"`
+	Type     string            `toml:"type"`
+	Address  string            `toml:"address"`
+	Platform string            `toml:"platform"`
+	Exports  map[string]string `toml:"exports"`
```

Comment:
Do we really need these? It's better to keep the existing configs for compatibility.

Reply:
We do need `Exports`, but we don't need `Platform`. 

I kept the `Platform` change because it was implemented before `Exports` and so I thought it would be cleaner (for a vague sense of the word "clean") to take both rather than just the newer piece that I needed.

I can remove the `Platform` backport if needed.